lib/minitest/mock.rb

```diff
@@ -198,6 +199,10 @@ class Object
   def stub name, val_or_callable, *block_args
     new_name = "__minitest_stub__#{name}"
 
+    if respond_to? new_name
+      raise StubError, "method :#{name.to_s} was already stubbed"
```

Comment:
If I'm not mistaken the interpolation already calls the `to_s` method. Was this not the case in your tests?